sds_data_manager/utils/stackbuilder.py

```diff
@@ -187,16 +187,21 @@ def build_sds(
         instrument_name="IalirtEcr",
     )
 
-    # TODO: look into creating a separate stackbuilder
-    #  for I-ALiRT
-    # I-ALiRT Processing (currently only IOIS)
-    ialirt_processing_stack.IalirtProcessing(
-        scope,
-        "IalirtProcessing",
-        env=env,
-        vpc=networking.vpc,
-        repo=ialirt_ecr.container_repo,
-    )
+    # All traffic to I-ALiRT is directed to listed container ports
+    ialirt_ports = {"Primary": [8080, 8081], "Secondary": [80]}
+    container_ports = {"Primary": 8080, "Secondary": 80}
+
+    for port in ialirt_ports:
```

Comment:
This is the string that is coming out, not the ports. The ports are in your lookup below.
```suggestion
    for primary_or_secondary in ialirt_ports:
```